pkg/controllers/csi/provisioner/controller.go

```diff
@@ -161,7 +161,7 @@ func (provisioner *OneAgentProvisioner) Reconcile(ctx context.Context, request r
 }
 
 func (provisioner *OneAgentProvisioner) setupFileSystem(dk *dynatracev1beta2.DynaKube) error {
-	tenantUUID, err := dk.TenantUUIDFromApiUrl()
+	tenantUUID, err := dk.TenantUUID()
```

Comment:
this will break the "resilience" feature, because
```
	err = provisioner.setupFileSystem(dk) // <= BOOM, because no tenant-uuidm because network is dead
	if err != nil {
		return reconcile.Result{}, err
	}
	tenantConfig, err := provisioner.setupTenantConfig(dk) // needed for the CSI-resilience feature
	if err != nil {
		return reconcile.Result{}, err
	}
```

Reply:
Set it back to the old method + follow up ticket created